src/ILCompiler.Compiler/src/CppCodeGen/CppWriter.cs

```diff
@@ -93,6 +90,21 @@ private void BuildExternCSignatureMap()
             }
         }
 
+        private IEnumerable<string> GetParameterNamesForMethod(MethodDesc method)
+        {
+            // TODO: The uses of this method need revision. The right way to get to this info is from
+            //       a MethodIL. For declarations, we don't need names.
```

Comment:
BTW: The new portable PDBs use the same format as ECMA metadata. The portable PDBs are conceptually just a few more metadata tables. I believe there is an option to store everything in the same file (though it is unlikely to be used much anytime soon because of none of the tooling understands it).